test/attr/attr_discardableResult.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-typecheck-verify-swift -enable-objc-interop
+// RUN: %target-typecheck-verify-swift -enable-objc-interop -disable-availability-checking
```

Comment:
Could this be moved to one of the concurrency test files instead?

Reply:
Yeah I thought about doing it, but the files in there pass this flag anyway so I thought I’d tweak this.

Reply:
That's okay is they do but at least it's all going to be grouped together...

Reply:
Alright done!

Reply:
Thank you!